regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/Suite.groovy

```diff
@@ -29,6 +29,7 @@ import org.apache.doris.regression.action.SuiteAction
 import org.apache.doris.regression.action.TestAction
 import org.apache.doris.regression.util.JdbcUtils
 import org.junit.jupiter.api.Assertions
+import org.slf4j.Logger
```

Comment:
We usually use 'org.apache.logging.log4j.Logger' in Doris ~

Reply:
That is ok, this is used for test framework.
And actually, we should unify Doris logger to slf4j.